app/models/user.rb

```diff
@@ -4,6 +4,16 @@ class User < ApplicationRecord
 
   attr_accessor :remember_token, :activation_token, :reset_token
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+                                  foreign_key: :follower_id,
```

Comment:
tại sao chỗ này phải đặt khóa phụ ông ơi

Reply:
khai báo foreign_key vì cần cho model biết nối đến cột nào bên model kia ô nhỉ, vì User tự many to many đến chính nó nên phải khai báo rõ ràng ra 2 trường liên kết với cột ID bên User đó chính là follower_id và followed_id